helper/helper_test.go

```diff
@@ -98,4 +189,60 @@ var _ = Describe("helper tests", func() {
 			Expect(isSubset).To(Equal(false))
 		})
 	})
+
+	Describe("doRestoreAgent Mocked unit tests", func() {
+		It("Test skip file in doRestoreAgent", func() {
```

Comment:
There is only one test case for DoRestore currently. If the intention to allow simple adding of unit tests in the future, I suggest to adjust the summary to be smth like: 'Add unit test infrastructure into gpbackup helper'.

And I suggest to add more unit tests for DoRestore in this commit (3-4 at least) - it could reveal if current approach is good or need to modify smth else.